app/assets/stylesheets/base/typography.scss

```diff
@@ -96,44 +95,52 @@ body {
   h5,
   h6 {
     font-family: var(--ff-sans-serif);
-    margin: var(--content-rythm) 0;
-    margin-top: calc(var(--content-rythm) * 2);
+    margin: calc(var(--content-rhythm) / 2) 0;
     padding: 0;
+    font-weight: var(--fw-bold);
+    line-height: var(--lh-tight);
   }
 
   h1 {
-    font-size: 1.85em;
-    line-height: 1.25em;
-    font-weight: 400;
+    font-size: 2em;
   }
 
   h2 {
-    font-size: 1.7em;
-    font-weight: 400;
-    line-height: 1.25em;
-    margin-bottom: 0.5em;
+    font-size: 1.8em;
   }
 
   h3 {
-    font-size: 1.32em;
-    font-weight: 400;
+    font-size: 1.6em;
+  }
+
+  h4 {
+    font-size: 1.4em;
+  }
+
+  h5 {
+    font-size: 1.2em;
+  }
+
+  h6 {
+    font-size: 1em;
```

Comment:
Do you think we can write these lines like this so that we can reference how they fit into our Crayons variables?

As someone who contributes to this code and and wants to maintain consistency, this would be very helpful.

`font-size: 1em // 18px // var(--fs-l);`

Reply:
`em` unit is relative to its parent and the parent font-size is dependent on two factors:

1. whether it's article body or article comments
2. breakpoint

So we can't really define what 1em means because we can't tell what's the main context :P. I think adding px values (or referencing variables) may add a bit of confusion in this case TBH.